index.html

```diff
@@ -27,6 +26,141 @@ <h1 class="section-header" id="my-name">Christian Onoh</h1>
+        <p class="see-project">See this project <i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>
+      <div class="project">
+        <img class="project-img" src="assets/images/pexels-eduardo-rosas-907487.jpg" alt="">
+        <header >
+          <h2 class="project-name">Project 2</h2>
+        </header>
+        <ul class="project-tools">
+          <li class="project-tool">HTML/CSS</li>
+          <li class="project-tool">Ruby on Rails</li>
+          <li class="project-tool">Javascript</li>
+        </ul>
+        <p class="see-project">See this project <i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>
+      <div class="project">
+        <img class="project-img" src="assets/images/pexels-ivan-samkov-5676744.jpg" alt="">
+        <header >
+          <h2 class="project-name">Project 3</h2>
+        </header>
+        <ul class="project-tools">
+          <li class="project-tool">HTML/CSS</li>
+          <li class="project-tool">Ruby on Rails</li>
+          <li class="project-tool">Javascript</li>
+        </ul>
+        <p class="see-project">See this project <i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>
+      <div class="project">
+        <img class="project-img" src="assets/images/pexels-lukas-574070 (1).jpg" alt="">
+        <header >
+          <h2 class="project-name">Project 4</h2>
+        </header>
+        <ul class="project-tools">
+          <li class="project-tool">HTML/CSS</li>
+          <li class="project-tool">Ruby on Rails</li>
+          <li class="project-tool">Javascript</li>
+        </ul>
+        <p class="see-project">See this project <i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>
+      <div class="project">
+        <img class="project-img" src="assets/images/pexels-mockupeditorcom-205316.jpg" alt="">
+        <header >
+          <h2 class="project-name">Project 5</h2>
+        </header>
+        <ul class="project-tools">
+          <li class="project-tool">HTML/CSS</li>
+          <li class="project-tool">Ruby on Rails</li>
+          <li class="project-tool">Javascript</li>
+        </ul>
+        <p class="see-project">See this project <i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>
+      <div class="project">
+        <img class="project-img" src="assets/images/pexels-pixabay-39284.jpg" alt="">
+        <header >
+          <h2 class="project-name">Project 6</h2>
+        </header>
+        <ul class="project-tools">
+          <li class="project-tool">HTML/CSS</li>
+          <li class="project-tool">Ruby on Rails</li>
+          <li class="project-tool">Javascript</li>
+        </ul>
+        <p class="see-project"> See this project<i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>      
+    </section>  
+    <section class="section-title">
+      <header>
+        <h1 class="section-header">About</h1>
+      </header>
```

Comment:
- [ ] I would suggest also changing the ```<h1>``` tag to another ```<hx>``` tag to follow good practices, we are recommended to use only one``` h1``` element per page. 😎😎 You may try to use other heading elements like **h2, h3, etc**. to create a clear hierarchy of headings and subheadings within the content.